src/main/scala/org/bitcoins/core/gen/NumberGenerator.scala

```diff
@@ -72,6 +72,8 @@ trait NumberGenerator {
 
   def scriptNumbers: Gen[ScriptNumber] = Gen.choose(Int64.min.underlying, Int64.max.underlying).map(ScriptNumber(_))
 
+  def unspendableLockTimeScriptNumbers : Gen[ScriptNumber] = Gen.choose(4294967296L, Int64.max.underlying).map(ScriptNumber(_))
```

Comment:
Add a comment on top of this function for what it is intended to be used for, is this meant for CSV, CLTV, both etc? Why are these numbers always unspendable?